examples/tutorials/core_api_pytorch_mnist/adaptive.yaml

```diff
@@ -28,7 +28,7 @@ searcher:
   name: adaptive_asha
   metric: test_loss
   smaller_is_better: true
-  max_trials: 500
+  max_trials: 50
   time_metric: epochs
   max_time: 20
-entrypoint: python3 model_def_adaptive.py
+entrypoint: python3 model_def_adaptive.py --epochs 20
```

Comment:
do you like this pattern of setting length in entrypoint? i get that the code for this is existing, but generally speaking.

i find it kind of annoying when iterating between local and on cluster modes, having two files to set max length in. is the reasoning that we don't want an extra if cluster_info/else in training code?

Reply:
> having two files to set max length in

which two files?

Reply:
i train locally, i change it in `train.py`. i want to submit it to cluster, i have to change `.yaml`